src/services/AddressService.jsx

```diff
@@ -0,0 +1,15 @@
+import _ from "lodash";
+
+const getFullAddress = address => {
+  return !_.isEmpty(address)
```

Comment:
this should be refactored, using object destructuring. not blocking, will submit another pr for this.